lib/heimdall_tools/help/sarif_mapper.md

```diff
@@ -0,0 +1,12 @@
+  sarif_mapper translates a SARIF JSON file into HDF format JSON to be viewable in Heimdall
+
+SARIF level to HDF impact Mapping:
+  SARIF level error -> HDF impact 0.7
+  SARIF level warning -> HDF impact 0.5
+  SARIF level note -> HDF impact 0.3
+  SARIF level none -> HDF impact 0.0
```

Comment:
This doc needs to be fixed given the last few commits

Reply:
Yes you are correct.